NAMESPACE

```diff
@@ -1,5 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(AffineAlignObj)
+export(AffineAlignObjLight)
+export(AffineAlignObjMedium)
+export(AlignObj)
```

Comment:
Can't find which changes added these into namespace file?

Reply:
I'm not sure, I didn't alter anything for these objects..?

Reply:
I will remove them from the file